helm/conjur-config-cluster-prep/templates/tests/test-cluster-prep-configmap.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-tests-configmap
+  annotations:
+    "helm.sh/hook": test
+    "helm.sh/hook-weight": "-1"
```

Comment:
Can you add a helpful reference to the Helm bug that explains why the `-1` hook weight is required into the PR description?

Reply:
Done